testing/util/block.go

```diff
@@ -1379,6 +1408,33 @@ func HydrateBeaconBlockBodyElectra(b *ethpb.BeaconBlockBodyElectra) *ethpb.Beaco
 	return b
 }
 
+// HydrateBeaconBlockBodyEPBS hydrates a beacon block body with correct field length sizes
+// to comply with fssz marshalling and unmarshalling rules.
+func HydrateBeaconBlockBodyEPBS(b *ethpb.BeaconBlockBodyEpbs) *ethpb.BeaconBlockBodyEpbs {
```

Comment:
This function should hydrate the important new fields like payload attestations and the signed execution payload header.